src/nodes.coffee

```diff
@@ -1695,14 +1713,38 @@ exports.CSXElement = class CSXElement extends Base
 
     {openingFragment, closingFragment}
 
+  contentAst: (o) ->
+    return [] unless @content and not @content.base.isEmpty?()
+
+    content = @content.unwrapAll()
+    children =
+      if content instanceof StringLiteral
+        [new CSXText content]
+      else # StringWithInterpolations
+        for element in @content.unwrapAll().extractElements o, includeInterpolationWrappers: yes
```

Comment:
We need the `Interpolation` location data for the `CSXExpressionContainer` (whose location data needs to include the surrounding `{`/`}` of the JSX expression), so added an `includeInterpolationWrappers` option to `StringWithInterpolations::extractElements()` (as well as a `locationData` option to `CSXExpressionContainer::constructor()`)